kuma/wiki/templates/wiki/feed_docs_updated.html

```diff
@@ -0,0 +1,12 @@
+<p>
+  <a href="{{ obj.parent.get_absolute_url()|absolutify }}" title="{{ obj.parent.title }}">
+    {% trans locale=obj.parent.locale %}View {locale} parent{% endtrans %}
+  </a>
+  (<a href="{{ mod_url|absolutify }}">{% trans modified=obj.parent.modified %}last modified at {modified}{% endtrans %}</a>)
+</p>
+<p>
+  <a href="{{ obj.get_edit_url()|absolutify }}" title="{{ obj.title }}">
```

Comment:
non-blocking scope-creep: Can we use the `add_utm` helper on the urls in this template? ([Like we do in the `plain.ltxt` template for welcome emails.](https://github.com/mozilla/kuma/blob/22390c1deec5ea40043871d8f20683cb642b5b2b/kuma/users/templates/users/email/welcome/plain.ltxt#L20-L39))

So, something like:
```
{{ add_utm(obj.get_edit_url()|absolutify, 'feed', medium=format) }}
```
Though I'm not sure you can call jinja output filters inside function arguments. :question: 

That way, GA can tell us how effective the feed links are at driving sessions that go on to translate docs, and we can compare that with emails to see which feature is more effective.